net/icmpv6/Kconfig

```diff
@@ -8,6 +8,7 @@ menu "ICMPv6 Networking Support"
 config NET_ICMPv6
 	bool "Enable ICMPv6 networking"
 	default n
+	select NET_MCASTGROUP
```

Comment:
should we change to depend?

Reply:
`NET_MCASTGROUP` is not selectable by menuconfig. https://github.com/apache/nuttx/blob/06e5b661ed6b1d7480b9c30a27be502a08924bce/net/Kconfig#L40-L42
And both `NET_IGMP` and `NET_MLD` select it to enable `d_addmac` and  `d_rmmac`. https://github.com/apache/nuttx/blob/06e5b661ed6b1d7480b9c30a27be502a08924bce/net/igmp/Kconfig#L6-L10
https://github.com/apache/nuttx/blob/06e5b661ed6b1d7480b9c30a27be502a08924bce/net/mld/Kconfig#L11
So I just let ICMPv6 keep the same as them.

Reply:
OK.